src/cmd/linuxkit/run_qemu.go

```diff
@@ -705,7 +655,7 @@ func buildQemuForwardings(publishFlags multipleFlag, containerized bool) (string
 func buildDockerForwardings(publishedPorts []string) ([]string, error) {
 	pmap := []string{}
 	for _, port := range publishedPorts {
-		s, err := splitPublish(port)
+		s, err := NewPublishedPort(port)
```

Comment:
the switch from `s` to `N` also means this is now a public function. Probably harmless but checking if it was intended and/or deemed ok.

Reply:
yes it was intended primarily because `DiskConfig` just above it is also public. But then some other functions in `util.go` are not, hey ho